tests/sklearn/test_metrics.py

```diff
@@ -101,3 +103,18 @@ def test_between_group_generalized_entropy_index():
     """Tests that the old and new between_group_GEI matches exactly."""
     bggei = between_group_generalized_entropy_error(y, y_pred, prot_attr='sex')
     assert bggei == cm.between_group_generalized_entropy_index()
+
+@pytest.mark.parametrize(
+    "func, is_ratio",
+    [
+        (statistical_parity_difference, False),
+        (disparate_impact_ratio, True),
+        (equal_opportunity_difference, False),
+        (average_odds_difference, False),
+    ],
+)
+def test_make_scorer(func, is_ratio):
+    actual = func(y, y_pred, prot_attr="sex")
```

Comment:
The test should also make sure if the `abs()` or `min()` condition applies. Maybe you could flip the protected attributes and make sure it's still equal?